frontend/src/app/page.tsx

```diff
@@ -1,5 +1,36 @@
 'use client';
 
+import styled from 'styled-components';
+import Text from '../design-system/Text';
```

Comment:
create import file and export the componet from there, it need to look like this
```suggestion
import Text from '@/design-system';
```